sdk/ai/Azure.AI.Inference/samples/Sample11_ImageEmbeddings.md

```diff
@@ -0,0 +1,80 @@
+# Simple Embeddings
+
+This sample demonstrates how to get embeddings for a provided image. Here we use the service default of returning embeddings as a list of floating point values.
+
+This sample assumes the AI model is hosted on a Serverless API or Managed Compute endpoint. For GitHub Models or Azure OpenAI endpoints, the client constructor needs to be modified. See package documentation for details.
+
+## Usage
+
+Set these two environment variables before running the sample:
+
+1. AZURE_AI_IMAGE_EMBEDDINGS_ENDPOINT - Your endpoint URL, in the form `https://your-deployment-name.your-azure-region.inference.ai.azure.com` where `your-deployment-name` is your unique AI Model deployment name, and `your-azure-region` is the Azure region where your model is deployed.
+
+2. AZURE_AI_IMAGE_EMBEDDINGS_KEY - Your model key. Keep it secret.
+
+```C# Snippet:Azure_AI_Inference_BasicImageEmbedding
+var endpoint = new Uri(System.Environment.GetEnvironmentVariable("AZURE_AI_IMAGE_EMBEDDINGS_ENDPOINT"));
+var credential = new AzureKeyCredential(System.Environment.GetEnvironmentVariable("AZURE_AI_IMAGE_EMBEDDINGS_KEY"));
+
+var client = new ImageEmbeddingsClient(endpoint, credential, new AzureAIInferenceClientOptions());
+
+List<ImageEmbeddingInput> input = new List<ImageEmbeddingInput>
+{
+    ImageEmbeddingInput.Load(imageFilePath:"sampleImage.png", imageFormat:"png")
```

Comment:
I don't see sampleImage.png as part of this PR. Can you add it to the samples folder?

Reply:
Based on Jesse's comment below, we don't want assets in this repo it seems. As it stands, these samples can't just be "run" like they can in python anyway, so I don't really have an issue with a user needing to fill in the path to the actual image they want to use. If we want to push for it though, we'll need to have a conversation with Jesse about being able to include the image in the repo

Reply:
We should be able to move it to the assets repository, along with the .mp3 file and test recordings.    @scbedd would be the best point of contact for getting them in there.

Reply:
Yeah, Scott has been helping me look into that for the test. It's not exactly a simple matter, as the test framework in C# evidently doesn't expose some of the information that would be the most useful to actually make this clean, but it should be doable with some work.

That's specifically for the test though. For this sample it doesn't really matter that the image (or audio file) is in the assets repo, I don't think we'd want to have a sample that shows some hacked around logic to get an asset from a different repo.

Reply:
That's unexpected.   I don't want to block on this; if we need to push into the repo in the short-term to keep things moving, I've no objections.   I'd just ask that we file an issue and loop back around to integrate into assets when possible.   A big downside to the mono repo approach is the size expanding over time, and the thought of binary files for tests scares me in that regard.

Reply:
Yeah, I understand. I'll make sure to keep track of it if I don't end up getting it working for this desired release.